src/Lime.Sample.AspNetCore/Lime.Sample.AspNetCore.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+    <PropertyGroup>
+        <TargetFramework>net5.0</TargetFramework>
+        <Nullable>enable</Nullable>
+        <LangVersion>8.0</LangVersion>
```

Comment:
langversion latest, we're almost at 10 already 😎 :p

Reply:
I prefer to leave the language version explicit, because "latest" is relative (my latest can be 7.0 and yours 10.0). But I will just remote this to keep it consistent with other projects.